src/util.h

```diff
@@ -49,6 +50,20 @@ NORETURN void Fatal(const char* msg, ...);
 #define NINJA_FALLTHROUGH
 #endif
 
+/// Whether C++11 features are enabled or not.
+#if __cplusplus >= 201103L || (defined(_MSC_VER) && _MSC_VER >= 1900)
+#  define NINJA_CPP11 true
+#else
+#  define NINJA_CPP11 false
+#endif
+
+template <typename T>
+T* to_address(T* ptr) { return ptr; }
+#if NINJA_CPP11
+template <typename T>
+decltype(auto) to_address(const std::unique_ptr<T>& ptr) { return ptr.get(); }
+#endif
```

Comment:
Minor, up to you if you want to do this,

but you could add a check for

#if defined([__cpp_lib_to_address](https://en.cppreference.com/w/cpp/utility/feature_test))

and if true, then replace these with `using std::to_address`, which is the same function, but as part of the standard.

Reply:
Cool. Thank you!